tasks/task.json

```diff
@@ -26,6 +26,24 @@
     "parameters": {
       "$ref": "#/definitions/paramsObject",
       "description": "An object mapping valid parameter names to corresponding json-schemas"
+    },
+    "implementations": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": ["name"],
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "requirements": {
+            "type": "array",
+            "additionalItems": {
+              "type": "string"
+            }
+          }
+        }
+      }
```

Comment:
great but i really do wish we had versioned schemas.

Reply:
We're discussing adding a revision number to the task spec that would cover that.